gusto/core/configuration.py

```diff
@@ -269,6 +269,20 @@ class BoundaryLayerParameters(Configuration):
     mu = 100.                   # Interior penalty coefficient for vertical diffusion
 
 
+class Held_Suarez_Parameters(Configuration):
+    """
+    Parameters used in the default configuration for the Held Suarez test case.
+    """
+    T0stra = 200      # Stratosphere temp
+    T0surf = 315      # Surface temperature at equator
+    T0horiz = 60      # Equator to pole temperature difference
+    T0vert = 10       # Stability parameter
+    sigmab = 0.7      # Height of the boundary layer
+    d = 24 * 60 * 60  # seconds in a day
```

Comment:
I think I'd prefer this not to be a parameter, and the tau values below can be e.g `40*24*60*60`